### ISSUE TITLE
Remove on-demand ClamAV option

### ISSUE THREAD

**Issue by swelljoe** [virtualmin/virtualmin-gpl#150]
ClamAV is so large and so slow to start today, that starting it on-demand is no longer reasonable. And, the language we've used around it "uses less RAM, slower mail processing" isn't reflective of reality (reality being that if someone is low on memory, using the on-demand mode will just cause the OOM killer to strike at random times, whenever mail arrives, rather than doing anything useful like freeing up memory when mail isn't being processed).

The Wizard should probably just offer to do AV scanning or not. If enabled, it uses `clamd` and if not, it just leaves ClamAV out of the picture entirely.

I've updated the language in the wizard to more accurately reflect reality, but it's probably worth simplifying the UI, too. Users are often confused by ClamAV and how much RAM it requires.

**Comment by iliarostovtsev** [comment]
I would also checked the amount of RAM available on that page in the wizard, and added a warning for a user, if it still wants to enable it on machines with low RAM.

**Comment by jcameron** [comment]
Yeah, that's a good point about the RAM spikes - I would support just dropping support for non-ClamAV mode.

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
